seshdash/views.py

```diff
@@ -782,13 +782,16 @@ def graphs(request):
             data_values = []
             time_delta = time_delta_dict[time]
             time_bucket=time_bucket_dict[time]
-            SI_unit = get_measurement_unit(choice)
+            SI_units = BoM_Data_Point.SI_UNITS
+            SI_unit = SI_units.get(choice,'V')		
+           # SI_unit = get_measurement_unit(choice)
+            print SI_unit
```

Comment:
let's remove print statments before PR. you can use logger.debug() if you think it's a useful debug message